### ISSUE TITLE
Customize the editor command

### ISSUE THREAD

**Issue by IndianBoy42** [marcusbuffett/pipe-rename#44]
This could be to use something other than `$EDITOR` for the renaming. But my use case is actually to set up some custom functionality in nvim, try and get better completion and highlighting.

Either way, a CLI argument for customizing the editor command.

maybe a config file in `XDG_CONFIG_HOME` for customizing the editor and rename command

**Comment by assarbad** [comment]
I'll look into the command line argument implementation _minus_ the config file topic. May take a few days.

**Comment by assarbad** [comment]
Once #59 is merged, this can be closed.

**Comment by assarbad** [comment]
@mtimkovich @marcusbuffett I think this could be closed now. Thanks.

If there is the desire/need for a configuration file, it'd make sense to split that off into a separate ticket anyway, don't you agree?
